### PR TITLE
[Docs] Add doc website

### PR DESCRIPTION
**Pull request by iiuhuy**
fiora-doc website by docusaurus.

### CODE REVIEW

**Comment by yinxin630** [comment]
Please provide some screenshots of doc website

File: website/blog/20210226-Blog-Test.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (yinxin630) [inline website/blog/20210226-Blog-Test.md]: Maybe should delete this test post?

File: website/languages.js [hunk 2]
```diff
@@ -0,0 +1,179 @@
+const languages = [
+    {
+        enabled: true,
+        name: 'English',
+        tag: 'en',
+    },
+    // {
```
Comment (yinxin630) [inline website/languages.js]: Delete these instead of commenting out

File: package.json [hunk 3]
```diff
@@ -29,7 +29,7 @@
     "npm": ">= 6.4.0"
   },
   "author": {
-    "name": "碎碎酱",
+    "name": " 碎碎酱 ",
```
Comment (yinxin630) [inline package.json]: Why are there extra spaces?

**Review by yinxin630 — CHANGES_REQUESTED** [review]

**Comment by iiuhuy** [comment]
> Please provide some screenshots of doc website

image

image

**Review by yinxin630 — APPROVED** [review]

**Pull request merged**